image-rauc.c

```diff
@@ -91,11 +96,14 @@ static int rauc_generate(struct image *image)
 			return ret;
 	}
 
-	systemp(image, "rm -f '%s'", imageoutfile(image));
+	if (keyring)
+		xasprintf(&keyringarg, "--keyring='%s'", keyring);
 
-	ret = systemp(image, "%s bundle '%s' --cert='%s' --key='%s' %s '%s'",
+	systemp(image, "rm -f '%s'", imageoutfile(image));
```

Comment:
You removed a newline here. Please add it again. It makes the diff a lot more readable.

Reply:
Done.